plugins/modules/nxos_l2_interfaces.py

```diff
@@ -37,12 +37,22 @@
 }
 
 DOCUMENTATION = """module: nxos_l2_interfaces
-short_description: Manages Layer-2 Interfaces attributes of NX-OS Interfaces
+short_description: L2 Interfaces Resource Module.
 description: This module manages Layer-2 interfaces attributes of NX-OS Interfaces.
 author: Trishna Guha (@trishnaguha)
 notes:
 - Tested against NXOS 7.3.(0)D1(1) on VIRL
 options:
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the NX-OS device by executing
+        the command B(show running-config | section ^interface).
+      - The state I(parsed) reads the configuration from C(running_config) option and transforms
+        it into Ansible structured data as per the resource module's argspec and the value is then
+        returned in the I(parsed) key within the result.
+    type: str
+    version_added: "1.0.0"
```

Comment:
I think this should be on the module itself.

Reply:
@rohitthakur2590 you are right, I'll be opening a separate PR for this change since this affects all nxos modules.

Reply:
@NilashishC  yeah that sounds good, as we will be having different sprint for that.